qiskit_nature/problems/sampling/protein_folding/peptide/beads/base_bead.py

```diff
@@ -67,7 +67,7 @@ def __init__(
         self._residue_type = residue_type
         _validate_residue_symbol(residue_type)
         self._turn_qubits = turn_qubits
-        if self._residue_type is not None and self.turn_qubits is not None:
+        if self._residue_type != "" and self.turn_qubits is not None:
```

Comment:
I don't have a strong opinion on this, but `if self._residue_type` is probably more Pythonic than `!= ""`